plugins/kotlin/idea/resources-en/messages/KotlinBundle.properties

```diff
@@ -2343,6 +2343,9 @@ fix.change.jvm.name=Change JVM name
 expand.boolean.expression.to.if.else=Expand boolean expression to 'if else'
 inspection.logger.initialized.with.foreign.class.display.name=Logger initialized with foreign class
 logger.initialized.with.foreign.class=Logger initialized with foreign class ''{0}''
+inspection.logger.placeholder.count.matches.argument.count.display.name=Number of placeholders does not match number of arguments in logging call
+placeholder.count.matches.argument.count.more.problem.descriptor=More arguments provided ''{0}'' than placeholders specified ''{1}''
```

Comment:
Both values for "More" and "Fewer" cases differ a bit from similar properties but of `PlaceholderCountMatchesArgumentCountInspection`.

Reply:
Thank you, now the messages are equal to PlaceholderCountMatchesArgumentCountInspection